### PR TITLE
created all the models

### PR DESCRIPTION
**Pull request by tuoanhnt95**

### CODE REVIEW

File: db/migrate/20220822081024_create_clinics.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateClinics < ActiveRecord::Migration[7.0]
+  def change
+    create_table :clinics do |t|
+      t.string :name
+      t.text :address
```
Comment (mengrui-song) [inline db/migrate/20220822081024_create_clinics.rb:5]: string maybe enough?

File: app/models/company.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class Company < ApplicationRecord
```
Comment (mengrui-song) [inline app/models/company.rb:1]: do we need:
has_many: users
has_many: set_ups

File: db/migrate/20220822081024_create_clinics.rb (hunk 1 above)
Comment (tuoanhnt95) [reply db/migrate/20220822081024_create_clinics.rb:5]: Okay, will change to string

File: app/models/company.rb (hunk 2 above)
Comment (tuoanhnt95) [reply app/models/company.rb:1]: Okay, will add now

**Pull request merged**